### PR TITLE
feat(files): add No Pale Oak Leaves Particles

### PR DESCRIPTION
**Pull request by SquatchHunter**
closes: #417 
using the same method as cherry leaves, we gut the particle generator, so it no longer spawns particles.

By checking the following boxes with an X, you ensure that:

- [x] The pack was tested ingame in at least one device.
- [x] The pack is an existing BT pack, is a missing pack from VT or is an accepted pack/change in a discussion.
- [x] The pack code follows the style guide.
- [x] The commits follow the contribution guidelines.
- [x] The PR follows the contribution guidelines.

- [x] (Optional) Tested in Windows
- [ ] (Optional) Tested in Android
- [ ] (Optional) Tested in iOS
- [ ] (Optional) Tested in any console
- [ ] (Optional) Tested in BDS

### CODE REVIEW

File: resource_packs/files/unobtrusive/no_pale_oak_leaves_particles/particles/pale_oak_leaves_particle.json
Comment (PAPISOP) [inline resource_packs/files/unobtrusive/no_pale_oak_leaves_particles/particles/pale_oak_leaves_particle.json:1]: The file should be named `pale_oak_leaves_particle.particle.json` to follow the style guide

**Review by PAPISOP — CHANGES_REQUESTED** [review]

File: resource_packs/files/unobtrusive/no_pale_oak_leaves_particles/particles/pale_oak_leaves_particle.json
Comment (drav0011) [reply resource_packs/files/unobtrusive/no_pale_oak_leaves_particles/particles/pale_oak_leaves_particle.json:1]: ^ other wise seems good

**Review by PAPISOP — APPROVED** [review]
lgtm

**Review by drav0011 — APPROVED** [review]
lgtm

**Pull request merged**